src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+export default class ErrorBoundary extends React.Component {

```

Comment:
Would be better to use `PureComponent` instead of `Component`. `Component` use only when you need to define `shouldComponentUpdate` method.